index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Project 1: Set up and Mobile Version Skeleton</title>
+
+  <link rel="stylesheet" type="text/css" href="styles.css">
+
+</head>
+<body>
+  <header>
+    <nav>
+      <ul class="nav-list">
+        <li><p id="nav-bar-p">Welcome</p></li>
```

Comment:
Kindly wrap the welcome with an `<a>` tag and not a `<p>`  because you have to place a link to the home page there to improve user experience